<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditClientDialog</class>
 <widget class="QDialog" name="EditClientDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Editar Cliente</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="labelAlignment">
      <set>Qt::AlignRight|Qt::AlignVCenter</set>
     </property>
     <property name="formAlignment">
      <set>Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <!-- Nombre completo -->
     <item row="0" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Nombre completo:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nameLineEdit"/>
     </item>
     <!-- DNI -->
     <item row="1" column="0">
      <widget class="QLabel" name="dniLabel">
       <property name="text">
        <string>DNI:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="dniLineEdit"/>
     </item>
     <!-- Email -->
     <item row="2" column="0">
      <widget class="QLabel" name="emailLabel">
       <property name="text">
        <string>Email:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="emailLineEdit"/>
     </item>
     <!-- Número de teléfono -->
     <item row="3" column="0">
      <widget class="QLabel" name="phoneLabel">
       <property name="text">
        <string>Número de teléfono:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="phoneLineEdit"/>
     </item>
     <!-- Dirección -->
     <item row="4" column="0">
      <widget class="QLabel" name="addressLabel">
       <property name="text">
        <string>Dirección:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="addressLineEdit"/>
     </item>
     <!-- Edad -->
     <item row="5" column="0">
      <widget class="QLabel" name="ageLabel">
       <property name="text">
        <string>Edad:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="ageSpinBox">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>150</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <!-- Botones de acción -->
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Guardar</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
